<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Bingo</class>
 <widget class="QMainWindow" name="Bingo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bingo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnGenerarCarton1">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Generar cartón 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGenerarCarton2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>200</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Generar cartón 2</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tblCarton2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>30</y>
      <width>261</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tblCarton1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>261</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBingo">
    <property name="title">
     <string>Bingo</string>
    </property>
   </widget>
   <addaction name="menuBingo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
